Expose fifoFilesFallbackPath as a config property. (#2235)
realm/realm-js: CONTRIBUTING.md

* Documentation is added in `docs/realm.js`
* Add an entry to `CHANGELOG.md` if applicable (Breaking changes/Enhancements/Bug fixes)

### Updating Object Store

> Object Store is the cross platform abstraction shared between all language bindings supported by Realm.

Reviewer: ```suggestion
Object Store is the cross platform abstraction shared between all SDKs supported by Realm.
```

> Object Store is the cross platform abstraction shared between all language bindings supported by Realm.

Reviewer: ```suggestion
[Object Store](https://github.com/realm/realm-object-store) is the cross platform abstraction shared between all language bindings supported by Realm.
```

> It is found [here](https://github.com/realm/realm-object-store).

Reviewer: ```suggestion
```

> It is included in Realm JS as a Git submodule under `/src/object-store`.

Reviewer: ```suggestion
It is included in Realm JS as a Git submodule under `src/object-store`.
```
